### PR TITLE
Use rollup

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -51,39 +54,34 @@
...
     "karma": "^6.3.2",
-    "karma-browserstack-launcher": "^1.6.0",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "^1.0.1",
     "karma-min-reporter": "^0.1.0",
     "karma-mocha": "^2.0.1",
     "karma-mocha-reporter": "^2.2.5",
+    "karma-rollup-preprocessor": "^7.0.8",
     "karma-safari-launcher": "^1.0.0",
     "karma-webpack": "^4.0.2",
```
Comment (tung-vu-td) [inline package.json:74]: Can we remove this yet?

File: rollup-dev.config.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+var commonjs = require('@rollup/plugin-commonjs')
+var nodeResolver = require('@rollup/plugin-node-resolve')
+
+module.exports = {
+  input: 'lib/index.js',
+  output: {
+    file: 'dist/td.js',
+    format: 'iife'
```
Comment (tung-vu-td) [inline rollup-dev.config.js:8]: TIL this acronym 😄

**Review by tung-vu-td — APPROVED** [review]
LGTM with one suggestion to remove karma-webpack

**Review by nmpennypacker — APPROVED** [review]
Reviewed for data exfiltrations

**Pull request merged**